src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private final int firstSide;
+    private final int secondSide;
+    private final FigureType figureType = FigureType.RECTANGLE;
```

Comment:
what if we move FigureType figureType field to Figure class?
each figure should have a type